packages/libs/react-ui/src/components/Form/Select/Select.css.ts

```diff
@@ -29,11 +29,7 @@ export const containerClassDisabled = style([
     },
   }),
   {
-    opacity: 0.4,
     selectors: {
-      '.inputGroup &': {
-        opacity: 1,
-      },
```

Comment:
I see this change is specifically for the disabled state. Did you test this? I think this is because the .inputGroup also applies the opacity change so this is making sure that lower opacity is not stacked

Took a look and I think it looks fine 👍

Reply:
![image](https://github.com/kadena-community/kadena.js/assets/73488793/be1bb56d-cd29-4246-bb8d-0b74d9168ba5)
yeah i did check and it was fine